src/sentry/api/endpoints/organization_releases.py

```diff
@@ -200,13 +201,16 @@ def get(self, request, organization):
         queryset = add_environment_to_queryset(queryset, filter_params)
 
         if query:
-            query_q = Q(version__icontains=query)
-
-            suffix_match = _release_suffix.match(query)
-            if suffix_match is not None:
-                query_q |= Q(version__icontains="%s+%s" % suffix_match.groups())
-
-            queryset = queryset.filter(query_q)
+            search_filters = parse_search_query(query)
+            # TODO: Handle semver here as well
+            for search_filter in search_filters:
+                if search_filter.key.name == RELEASE_FREE_TEXT_KEY:
```

Comment:
Is `FREE_TEXT` right here, isn't it more like a text filter? Free text is when it's text outside of the filter `key:val` format

Reply:
Nevermind I misread.